pkg/asset/manifests/azure/cluster.go

```diff
@@ -77,24 +84,85 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 						Type: capz.Internal,
 					},
 				},
+				ControlPlaneOutboundLB: &capz.LoadBalancerSpec{
+					FrontendIPsCount: to.Ptr(int32(1)),
+				},
 				Subnets: capz.Subnets{
 					{
 						SubnetClassSpec: capz.SubnetClassSpec{
-							Name: "control-plane-subnet",
+							Name: controlPlaneSubnet,
 							Role: capz.SubnetControlPlane,
 							CIDRBlocks: []string{
 								subnets[0].String(),
 							},
 						},
+						SecurityGroup: capz.SecurityGroup{
+							Name: controlPlaneNSG,
```

Comment:
Will this `Name` field allow us to set the name of the nsg that capz creates? If so it seems it would be better to set this to our existing naming convention.

Reply:
Updated to use a single NSG.